### PR TITLE
Version changed to v0.20.0

### PR DESCRIPTION
**Pull request by spirali**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@
   The new architecture also allows parallel I/O writing and storing of multiple job streams in one stream handle.
   You can use worker streaming using the `--stream` parameter of `hq submit`. Check out the documentation for more information.
 
+* Optimization of journal size
```
Comment (Kobzol) [inline CHANGELOG.md:16]: Maybe we could mention/remind that dashboard is still disabled.

**Review by Kobzol — APPROVED** [review]
Left one nit, otherwise LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (spirali) [reply CHANGELOG.md:16]: It is a status quo, and this should be a changelog, isnt it?:)

Comment (Kobzol) [reply CHANGELOG.md:16]: I know, I know :) But some people might have expected that the next version will enable it again.

Comment (spirali) [reply CHANGELOG.md:16]: Ok, I am adding a note

**Pull request merged**